content/tracing/getting_further/first_class_dimensions.md

```diff
@@ -65,32 +65,45 @@ There are several ways to specify an environment when reporting data:
 3. Per trace:  
   When submitting a single trace, specify an environment by tagging one of its spans with the metadata key `env`. This overrides the Agent configuration and the host tags value (if any).  
 
-  * **Go**:
-    ```
-    tracer.SetTag("env", "prod")
-    ```
-  For OpenTracing use the `tracer.WithGlobalTag` start option to set the environment globally.
-
-  * **Java**:
-      Via sysprop:
-      ```
-      -Ddd.trace.span.tags=env:prod
-      ```
-      Via env var:
-      ```
-      DD_TRACE_SPAN_TAGS="env:prod"
-      ```
-
-  * **Ruby**:
-  ```
-  Datadog.tracer.set_tags('env' => 'prod')
-  ```
-
-  * **Python**:
-    ```
-    from ddtrace import tracer
-    tracer.set_tags('env', 'prod')
-    ```
+{{< tabs >}}
+{{% tab "Go" %}}
+
+```go
+tracer.SetTag("env", "prod")
+```
+
+For OpenTracing use the `tracer.WithGlobalTag` start option to set the environment globally.
+
+{{% /tab %}}
+{{% tab "Java" %}}
+Via sysprop:
+
+```
+-Ddd.trace.span.tags=env:prod
+```
+
+Via env var:
```

Comment:
I kind of want to spell this out and say "Via environment variables:"
Thoughts?

Reply:
agreed :)